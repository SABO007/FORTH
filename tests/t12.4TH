: SQUARE DUP * ;
5 SQUARE .
10 SQUARE .
2.5 SQUARE .