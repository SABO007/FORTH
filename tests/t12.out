25
100
6.25
